### ISSUE TITLE
/opt/conda/lib/python3.7/site-packages/graphql/type/directives.py:55: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working   assert isinstance(locations, collections.Iterable), 'Must provide locations for directive.

### ISSUE THREAD

**Issue by williamFalcon** [allegroai/trains#155]
Hi!

This warning comes up very frequently and i think it's because of this library.
Could you guys take a look at updating these calls?

Thank you!

```
/opt/conda/lib/python3.7/site-packages/graphql/type/directives.py:55: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
  assert isinstance(locations, collections.Iterable), 'Must provide locations for directive.
```

**Comment by bmartinn** [comment]
Hi @williamFalcon ,

I think this issue has already been addressed by #110 and fixed in #111 

Trains does not import graphql, which seems like the root cause of this warning, or am I missing something here?

**Comment by bmartinn** [comment]
Seems like this issue was addressed a while ago.
